salt/fileserver/roots.py

```diff
@@ -130,27 +130,36 @@ def update():
     mtime_map_path = os.path.join(__opts__['cachedir'], 'roots/mtime_map')
     # data to send on event
     data = {'changed': False,
+            'files': {'changed': []},
```

Comment:
They are always going to be initialized further on when we use the sets to calculate what has been added/removed. It may render an empty list, therefore, accomplishing the initialization.

I'd like to always have that on the event so the document is predictable and no check on the changed element needs to be done when iterating over the event on SLS files, for example.